packages/db/src/models/execution.ts

```diff
@@ -34,6 +34,28 @@ class ExecutionPgModel extends BasePgModel<PostgresExecution, PostgresExecutionR
       .merge()
       .returning('cumulus_id');
   }
+
+  /**
+   * Get workflow_name column values from the execution cumulus_id
+   *
+   * @param {Knex | Knex.Transaction} knexOrTrx -
+   *  DB client or transaction
+   * @param {Function} executionCumulusIds -
+   * single execution cumulus_id or array of exeuction cumulus_ids
+   * @returns {Promise<Array<number>>} An array of workflow names
+   */
+  async getWorkflowNamesFromExecutionCumulusIds(
+    knexOrTrx: Knex | Knex.Transaction,
+    executionCumulusIds: Array<string> | string
+  ): Promise<Array<string>> {
+    const executionCumulusIdsArray
+      = (typeof executionCumulusIds === 'string') ? [executionCumulusIds] : executionCumulusIds;
```

Comment:
If we're sure we have either a string or an array, which we should be because it's Typescript, then you could do

`const executionCumulusIdsArray = [executionCumulusIds].flat();`

Reply:
Oh neat! Done